packages/core/src/index.js

```diff
@@ -135,7 +140,7 @@ class Editor {
 
   setContentPlugins = (plugins: Array<any> = []) => {
     this.plugins.setContentPlugins(plugins)
-    console.log(this.store.getState())
+    console.log(this.getState())
```

Comment:
This can be removed

Reply:
Sure, theres other console.logs that probs shouldn't be there either.